.github/workflows/documentation_checks.yml

```diff
@@ -0,0 +1,19 @@
+name: Documentation checks
+
+on:  
+  pull_request:
+    branches: 
+      - main
+    paths:
+      - 'docs/**'  
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true  
+
+jobs:
+  documentation-checks:
+    uses: canonical/documentation-workflows/.github/workflows/documentation-checks.yaml@main
```

Comment:
It's nice that we can re-use these workflows like this although I have slight concern that be referring to main changes made in that repo can mean that we see variability in the output of checks on this project.

Do you know if there are plans to release/tag the workflows so there is confidence they are stable?

Reply:
This is indeed a valid concern. I opened an issue in the original repo (https://github.com/canonical/documentation-workflows/issues/6) to call for attention. 
Meanwhile, since the referenced scripts are not complex, I changed to the hard-coded version in the new commit unless more advanced features are introduced.